File: 0058-length-of-last-word/0058-length-of-last-word.py
class Solution(object):
    def lengthOfLastWord(self, s):
        c = 0
        for i in range (len(s)-1, -1, -1):
            if (s[i].isalpha()):
                c += 1
            elif (s[i] == ' ' and c != 0):
                break
        return c           

File: 0058-length-of-last-word/0058-length-of-last-word.java
class Solution 
{
    public int lengthOfLastWord(String s) 
    {
        int c = 0;
        for (int i = s.length() - 1; i >= 0; i--)
        {
            if (Character.isLetter(s.charAt(i)))
            {
                c++;
            }
            else if (s.charAt(i) == ' ' && c != 0)
            {
                break;
            }
        }
        return c; 
    }
}